docs/decisions/0002-auto-enrollment-post-request-approval.rst

```diff
@@ -0,0 +1,158 @@
+2. Automatically Enrolling Learner After a Subsidy Request Approval
+===================================================================
+
+
+Status
+======
+
+?
+
+Context
+=======
+
+Learners are currently able to request a subsidy (license or coupon code) through the enterprise-access service.
+After a request is approved, the learner has to redeem the subsidy and enroll in a course manually.
+Since the goal of a learner is to access a course, we want to automatically enroll them after a request approval.
+
+For coupon code requests, requiring manual redemption is not ideal because we cannot enforce that a code is only used
+for the course it was requested for.
+Admins have some visibility into the usage of a coupon code via the admin portal. It shows only if a coupon code has been redeemed,
```

Comment:
"it shows only if" What is the it? The request? The code? Be more specific. Perhaps "The code only shows if it has been redeemed, but not the course for which it was redeemed." 

(FWIW I'm not a fan of dangling participles)